### PR TITLE
MSTeams Bot Updates

### PR DESCRIPTION
**Pull request by CalvinAllen**
Added NR.com to validDomains
Removed extraneous messages from bot
Incremement version for testing

### CODE REVIEW

File: api_server/etc/msteamsbot/template/manifest.json [hunk 1]
```diff
@@ -83,6 +83,7 @@
         "*.codestream.us",
         "csaction.cdstrm.us",
         "*.cdstrm.us",
-        "*.cdstrm.dev"
+        "*.cdstrm.dev",
+        "*.*.newrelic.com"
```
Comment (CalvinAllen) [inline api_server/etc/msteamsbot/template/manifest.json:87]: I added this before really checking, but I don't think we need ANY of these listed. The docs say this node is for when your bot "expects to load the website within the teams client" - which we don't do.

File: api_server/modules/providers/msteams_conversation_bot.js [hunk 2]
```diff
@@ -640,7 +640,6 @@ class MSTeamsConversationBot extends TeamsActivityHandler {
 		});
 
 		await this.sendAdaptiveCard(context, body);
-        await context.sendActivity('After signing in, please copy the code shown on your screen and paste it here.');
```
Comment (CalvinAllen) [inline api_server/modules/providers/msteams_conversation_bot.js:643]: This line is only supposed to exist for the `helpPersonal` command - we don't want users pasting their tokens into the Channel chat

File: api_server/modules/providers/msteams_conversation_bot.js [hunk 3]
```diff
@@ -721,7 +720,6 @@ class MSTeamsConversationBot extends TeamsActivityHandler {
...
```
Comment (CalvinAllen) [inline api_server/modules/providers/msteams_conversation_bot.js:724]: This line is only supposed to exist for the `helpPersonal` command - we don't want users pasting their tokens into the Channel chat

File: api_server/etc/msteamsbot/template/manifest.json (hunk 1 above)
Comment (cstryker) [reply api_server/etc/msteamsbot/template/manifest.json:87]: sure, but i guess just keep it in anyway, no harm

**Review by cstryker — APPROVED** [review]

**Pull request merged**